### DISCUSSION TITLE
Style classes behavior

### DISCUSSION THREAD

**Discussion by georgweiss** [ControlSystemStudio/phoebus#2324] [General]
Currently if user selects a "style" class, it looks like the properties defined in that class cannot be edited. Is this intentional/wanted behavior?

**Comment by kasemir** [comment]
That is exactly how it's supposed to work, see https://controlssoftware.sns.ornl.gov/training/2022_USPAS/Presentations/05%20Display%20Builder.pdf page 47.
Properties set by the class are, well, set by the class so you cannot change them.
You're free to not use the class, then you can edit all properties.

**Reply by evasann** [reply]
I cannot see the document you are linking to, can you provide it in full? 
Or send it to me/us in an email?

**Comment by georgweiss** [comment]
OK, thanks.

**Comment by kasemir** [comment]
You can't see https://controlssoftware.sns.ornl.gov/training/2022_USPAS ??

**Reply by evasann** [reply]
I can see it now, thanks.
For some reason I got a 404.
